README.md

```diff
@@ -55,7 +55,7 @@ import ChipInput from 'material-ui-chip-input'
 | openOnFocus | `bool` | `false` | Opens the auto complete list on focus if set to true. |
 | style | `object` | | Override the inline-styles of the root element. |
 | value | `string[]` | | The chips to display (enables controlled mode if set). |
-
+| allowDuplicates | `bool` | | If true allows duplicate chips (default false) |
```

Comment:
Please use the style of the other props and keep the alphabetical order (the default value belongs into the third column and the description should end with a dot).